compiler/testData/codegen/bytecodeListing/annotations/localClassWithCapturedParams_ir.txt

```diff
@@ -0,0 +1,25 @@
+@kotlin.Metadata
```

Comment:
Diff in this test:
```
annotation class Simple(val value: String)

fun localCaptured(): Any {
    val z  = 1
    class A(@Simple("K") val z: String) {
        val x = z // no longer creates `final synthetic String $z`
    }
    return A("K")
}
```
The fact that `z` refers to the local `z` and not the property `z` looks questionable to me, but that's not the point right now.